src/libaktualizr/package_manager/CMakeLists.txt

```diff
@@ -17,9 +17,6 @@ add_aktualizr_test(NAME packagemanagerfake SOURCES packagemanagerfake_test.cc)
 # Debian backend
 if(BUILD_DEB)
     set_property(SOURCE packagemanagerfactory.cc packagemanagerfactory_test.cc PROPERTY COMPILE_DEFINITIONS BUILD_DEB)
```

Comment:
We probably don't need to set this `BUILD_DEB` definition here anymore, right? In fact, is `BUILD_DEB` meaningful at all anymore? In CI it's built by default, and if it brings in no dependencies, there's hardly a reason not to just have it always on.

Reply:
That's a fair point, though we'd ship a code path that calls dpkg on yocto systems where it doesn't make sense. I don't know if the cost of keeping it (low, I think) warrants larger executables and potential bug surface.

Reply:
Probably true. Perhaps we should disable by default on CI and only enable it as necessary?

Reply:
Our approach on CI has been to enable all flags, so that everything can be tested at once. The things that are not covered tend to not compile after refactors.

Also, some unrelated useful tests use the Debian mode. For example memory_test. We'll probably migrate them when we can though.

Reply:
Fair enough. I have no objection.

Reply:
@patrickvacek, @lbonn Sorry, I'm a bit confused, what's your conclusion on that?

Reply:
I think just keep it as is?

Reply:
Cool, that's what I thought, just wanted to be sure.

Reply:
Yes, sorry for the distraction!